functionalDB/chapter.md

```diff
@@ -1,103 +1,104 @@
-# Designing a database like an archaeologist
+<!-- American spelling --> 
+# CircleDB: An Archaeology-Inspired Database

```

Comment:
I'll leave the final decision up to @MichaelDiBernardo --- it will depend on the amount of available space in the table of contents, etc., as well as whether other book chapters will have subtitles and if so, how they are formatted.